ui/docs/reference/core/worker-manager/worker-interaction.mdx

```diff
@@ -48,6 +48,16 @@ It is the responsibility of the worker to finish its work before the credentials
 
 Depending on the provider, `registerWorker` may be called multiple times for the same worker, or may only be called once.
 
+## Reregistration
+
+The worker may reregister itself to renew its credentials with the worker manager by calling
+[`workerManager.reregister`](/docs/reference/core/worker-manager/api#reregisterWorker) and
+supplying it with a one time usage `secret`. The secret value will have been either last configured
+in `registerWorker` (in the case of a newly registerd worker) or `reregisterWorker`.
+The expiration time of the new credentials can be controlled via `reregistrationTimeout` which gets set when
+calling [`workerManager.registerWorker`](/docs/reference/core/worker-manager/api#registerWorker).
+The default behavior, when `reregistrationTimeout` is omitted, is to extend the credentials by 96 hours.
```

Comment:
The value of this actually comes from workerpool configuration right? The call to `reregisterWorker` can't change this field iiuc.

Reply:
Correct. I went ahead and rephrased it to say that it comes from the worker pool configuration. Happy to change it to something else.